ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -15,7 +15,15 @@
 module D = Debug.Make(struct let name = "xapi" end)
 open D
 
-open Stdext.Xstringext
+open Stdext
```

Comment:
I think if we don't use `List.filter_map` (see below) then there is no need to open these additional modules.